stl/src/xtime.cpp

```diff
@@ -8,10 +8,10 @@
 #include <time.h>
 #include <xtimec.h>
 
-#define NSEC_PER_SEC  1000000000L
-#define NSEC_PER_MSEC 1000000L
-#define NSEC_PER_USEC 1000L
-#define MSEC_PER_SEC  1000
+constexpr long NSEC_PER_SEC = 1000000000L;
+constexpr long NSEC_PER_MSEC = 1000000L;
+constexpr long NSEC_PER_USEC = 1000L;
+constexpr int MSEC_PER_SEC = 1000;

```

Comment:
Any commits added to the branch you submitted for this pull request (PR) are automatically added to the PR. This is convenient when you're making changes in response to review comments and you *want* those changes to appear in the PR, but not so much when you're working on other changes that aren't ready to merge or shouldn't be merged as part of the same PR.

Reply:
(If these changes are intended to be part of this PR, your description should say so and provide some rationale.)

Reply:
oh okay , I see so let me revert then